pydantic_extra_types/payment.py

```diff
@@ -90,37 +98,56 @@ def validate_brand(card_number: str) -> PaymentCardBrand:
         Validate length based on BIN for major brands:
         https://en.wikipedia.org/wiki/Payment_card_number#Issuer_identification_number_(IIN)
         """
+        brand = PaymentCardBrand.other
+
         if card_number[0] == '4':
             brand = PaymentCardBrand.visa
+            required_length = [13, 16, 19]
         elif 51 <= int(card_number[:2]) <= 55:
             brand = PaymentCardBrand.mastercard
+            required_length = [16]
         elif card_number[:2] in {'34', '37'}:
             brand = PaymentCardBrand.amex
+            required_length = [15]
         elif 2200 <= int(card_number[:4]) <= 2204:
             brand = PaymentCardBrand.mir
-        else:
-            brand = PaymentCardBrand.other
-
-        required_length: None | int | str = None
-        if brand in PaymentCardBrand.mastercard:
-            required_length = 16
-            valid = len(card_number) == required_length
-        elif brand == PaymentCardBrand.visa:
-            required_length = '13, 16 or 19'
-            valid = len(card_number) in {13, 16, 19}
-        elif brand == PaymentCardBrand.amex:
-            required_length = 15
-            valid = len(card_number) == required_length
-        elif brand == PaymentCardBrand.mir:
-            required_length = 'in range from 16 to 19'
-            valid = len(card_number) in range(16, 20)
-        else:
-            valid = True
+            required_length = list(range(16, 20))
+        elif card_number[:4] in {'5018', '5020', '5038', '5893', '6304', '6759', '6761', '6762', '6763'}:
+            brand = PaymentCardBrand.maestro
+            required_length = list(range(12, 20))
+        elif card_number[:4] in {'6759', '676770', '676774'}:
```

Comment:
`card_number[:4] == '6759' or card_number[:6] in ('676770', '676774')`

Reply:
Thank you @dimastbk